src/sqlite/gnss.ts

```diff
@@ -3,11 +3,11 @@ import { db } from './index';
 import { convertTimestampToDbFormat } from 'util/index';
 
 export const fetchGnssLogsByTime  = async (from: number, to?: number): Promise<GnssRecord[]> => {
-    let query = `SELECT * FROM gnss WHERE time > ?`;
+    let query = `SELECT * FROM gnss WHERE system_time > ?`;
```

Comment:
@mazar-farran Motion model expects this part to select based on `time`,
as a reliable metrics of a real current time, so let's bring it back to `time`, if you don't mind!

Reply:
Done, changed back to `time`